src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -110,19 +111,19 @@ def get_vm_client_with_shorter_polling_interval(_):
 factory = lambda _: get_mgmt_service_client(ACSClient).acs
 cli_command('acs create', AcsOperations.create_or_update, factory, transform=DeploymentOutputLongRunningOperation('Starting container service create'))
 
-factory = lambda _: _compute_client_factory().container_service
+factory = lambda _: _compute_client_factory().container_services
 #Remove the hack after https://github.com/Azure/azure-rest-api-specs/issues/352 fixed
 from azure.mgmt.compute.models import ContainerService#pylint: disable=wrong-import-position
 for a in ['id', 'name', 'type', 'location']:
     ContainerService._attribute_map[a]['type'] = 'str'#pylint: disable=protected-access
 ContainerService._attribute_map['tags']['type'] = '{str}'#pylint: disable=protected-access
 ######
-cli_command('acs show', ContainerServiceOperations.get, factory)
-cli_command('acs list', ContainerServiceOperations.list, factory)
-cli_command('acs delete', ContainerServiceOperations.delete, factory)
+cli_command('acs show', ContainerServicesOperations.get, factory)
+cli_command('acs list', list_container_services)
```

Comment:
Change this to `cli_command('acs list', list_container_services, factory)` as line 114 already createes to container_services client.